packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 ### New features and enhancements
 
+- Marked `protected _lookupParentDirectory()` method as `@deprecated` and redefined it as a `public lookupParentDirectory()` method. [#3485] (https://github.com/zowe/zowe-explorer-vscode/issues/3485)
```

Comment:
Is it possible just to say, "Deprecated the `bbbb` method and redefined it as the `zzzz` method."?